rotkehlchen/errors.py

```diff
@@ -123,3 +123,11 @@ class ModuleInitializationFailure(Exception):
 
 class ModuleInactive(Exception):
     """Raises when call happens for a module that is not initialized"""
+
+
+class WriteError(Exception):
+    """Raises when there is an error writing a file to disk"""
+
+
+class InvalidData(Exception):
+    """Raises when there is an error writing a file to disk"""
```

Comment:
you copy pasted the error message from the previous exception here. Also no need to create a new error. We got one: `UnableToDecryptRemoteData`.